### ISSUE TITLE
Circular docs loops for config

### ISSUE THREAD

**Issue by zaccb** [conventional-changelog/conventional-changelog#682]
There's a circular documentation loop that prevents users from getting access to configuration docs for this repo.

When you go to the top-level repo, you are forced into one of the packages for configuration guidance. From the CLI package, you are then directed back to the top-level repo instead of provided with resources for how to configure the CLI.

The link in the README for the CLI docs is: https://github.com/conventional-changelog/conventional-changelog

Where is the actual location of the CLI-specific configuration documentation? If we know where it is, I am happy to fire a PR and update it so that users of the CLI package aren't redirected in an infinite loop and don't have to chain their way through many repros to get config information.

**Comment by shaunc** [comment]
Indeed! In particular where is the documentation for `changelog.config.js`? It used to exist somewhere, I'm sure....

**Comment by dangreen** [comment]
Configuration is now documented separately in the CLI and API guides, including config files and preset customization. Closing as resolved.
